Makefile

```diff
@@ -2,11 +2,13 @@ LIBVIRT_HOOKS_DIR=/etc/libvirt/hooks
 
 install:
 	mkdir -p ${LIBVIRT_HOOKS_DIR}
-	cp qemu qemu.schema.json ${LIBVIRT_HOOKS_DIR}
-	if [ ! -f ${LIBVIRT_HOOKS_DIR}/qemu.json ] ; then cp qemu.json ${LIBVIRT_HOOKS_DIR} ; fi
-	chmod +x ${LIBVIRT_HOOKS_DIR}/qemu
-	service libvirtd restart
+	cp hooks hooks.schema.json ${LIBVIRT_HOOKS_DIR}
+	if [ ! -f ${LIBVIRT_HOOKS_DIR}/hooks.json ] ; then cp hooks.json ${LIBVIRT_HOOKS_DIR} ; fi
+	chmod +x ${LIBVIRT_HOOKS_DIR}/hooks
+	ln -s ${LIBVIRT_HOOKS_DIR}/hooks ${LIBVIRT_HOOKS_DIR}/qemu
+	ln -s ${LIBVIRT_HOOKS_DIR}/hooks ${LIBVIRT_HOOKS_DIR}/lxc
```

Comment:
Yes, symlinks do the job too. Depending on the libvirt domain it is executed, it runs qemu or lxc hooks, however those network hooks migth apply for both.